README.md

```diff
@@ -21,24 +25,24 @@
 
 - `livekit-core`: LiveKit protocol implementation
 - `livekit-utils`: Shared utilities between our crates
-- `livekit-ffi`: Use `livekit-core` on foreign languages
+- `livekit-ffi`: Bindings for other languages. use `livekit-core`.
```

Comment:
Is there a typo here, or it is just my English ?😀

Reply:
naw no typos, sounds more natural to call it bindings.